bin/currency/update-prod-dependencies.js

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Putting it here: Naming problem

bin/currency/update-currencies.js
bin/currency/update-prod-dependencies.js

Please come up with a better strukture.